docs/blog/gatsbygram-case-study/index.md

```diff
@@ -374,7 +374,7 @@ generates all the configuration for you.
 
 Normally page resources are pre-cached with a service worker. But as several
 browsers (Safari/Microsoft Edge) still don't support Service Workers, the
-[Gatsby `<Link>` component](/packages/gatsby/) pre-caches resources for
+[Gatsby `<Link>` component](https://www.gatsbyjs.org/docs/linking-between-pages/#the-gatsby-link-component) pre-caches resources for
```

Comment:
It's not obvious, but these should be relative links: 

```suggestion
[Gatsby `<Link>` component](/docs/linking-between-pages/#the-gatsby-link-component) pre-caches resources for
```

Reply:
Okkk! I got it!